src/app/page.tsx

```
@@ -1,119 +1,51 @@
-              >
-                ✕
-              </button>
-            </div>
-          ))}
+        {/* Botón de ejemplo */}
+        <div className="flex justify-center mt-6">
+          <button 
+            onClick={() => setThemeColor(`hsl(${Math.random() * 360}, 70%, 60%)`)}
+            className="bg-white/20 hover:bg-white/30 text-white px-6 py-3 rounded-lg transition-colors"
+          >
+            Cambiar Color
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)